test/integration/pack/test_pack.py

```diff
@@ -1759,9 +1759,9 @@ def test_no_cache_yml_file(cartridge_cmd, light_project, tmpdir, pack_format):
         cmd.append('--use-docker')
 
     rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
-    assert rc == 1
+    assert rc == 0
     assert "Using cached path" not in output
-    assert f"File {os.path.join(project.path, 'pack-cache.yml')} with pack cache parameters doesn't exists"
+    assert "Failed to process pack-cache-config.yml file which contain cache paths" in output
```

Comment:
I don't think that we should show this when the file doesn't exist.
If it exists, but stat failed - it's an error, somebody created this file but it's not possible to use it. If it doesn't exist - we don't need the cache.